data/json/recipes/armor/storage.json

```diff
@@ -51,6 +51,20 @@
     "qualities": [ { "id": "SEW", "level": 1 } ],
     "components": [ [ [ "leather", 5 ], [ "rag", 5 ] ], [ [ "chestpouch", 4 ], [ "legpouch_large", 2 ] ], [ [ "filament", 20, "LIST" ] ] ]
   },
+  {
+    "result": "axe_ring",
+    "type": "recipe",
+    "category": "CC_ARMOR",
+    "subcategory": "CSC_ARMOR_STORAGE",
+    "skill_used": "tailor",
+    "difficulty": 4,
+    "skills_required": [ "melee", 1 ],
```

Comment:
I wonder if melee combat skill is really needed for crafting anything.

Reply:
It's what other melee holster have. Baldrics, sheaths, scabbards, and back scabbards have it so I thought I might add it as well.

Reply:
I'd prefer removing the requirement of melee skill from those recipes instead of adding it to another one.

Reply:
I'll remove that requirement on the other items if it's ok, tho it's kind of stepping outside my intended PR.

Reply:
It should be done in a different PR, it's unrelated to this one.